### PR TITLE
Map Markers Feature Added (Issue #8)

### PR DESCRIPTION
**Pull request by ayush221b**
This solves issue #8 

The demo makes use of <a href="https://leafletjs.com/" target="_blank"> Leafletjs </a>, and <a href="https://www.openstreetmap.org/" target="_blank"> OpenStreetMaps API </a>. The API is completely open source and free to use.

The data is present in an array of objects, in the chapterMapConfig.js file, and can be very easily modified. Also, the markers can be customised if required. For a full demo, and better review, please visit: https://gssoc-eff9f.firebaseapp.com/

### CODE REVIEW

File: poc/feature-mapmarkers/ayush221b/index.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Map Markers Demo for GirlScript India
+        <</title>
```
Comment (vishwarajanand) [inline poc/feature-mapmarkers/ayush221b/index.html]: Should be just `<`, right?

File: poc/feature-mapmarkers/ayush221b/index.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+            var markers = chapters.map(chapter => {
+                return (L.marker([Number(chapter.latitude), Number(chapter.longitude)]).addTo(
+                            mymap)
+                        .bindTooltip(`${chapter.city} GirlSCript`).openTooltip())
+                        .on('click', () => {
+                            window.open(chapter.website)
+                        });
+            })
+        </script>
+    </div>
```
Comment (vishwarajanand) [inline poc/feature-mapmarkers/ayush221b/index.html:44]: Can you add screenshots also?

File: poc/feature-mapmarkers/ayush221b/index.html (hunk 1 above)
Comment (ayush221b) [reply poc/feature-mapmarkers/ayush221b/index.html]: Yes, fixing it now.

File: poc/feature-mapmarkers/ayush221b/index.html (hunk 2 above)
Comment (ayush221b) [reply poc/feature-mapmarkers/ayush221b/index.html:44]: Yes.

**Comment by ayush221b** [comment]
@vishwarajanand I have added the screenshots and fixed the redundant '<'. For a full demo, please visit: https://gssoc-eff9f.firebaseapp.com/ .
Thanks.

**Comment by vishwarajanand** [comment]
LGTM

**Pull request merged**